docker/sds-api/spine-directory-service/sds/request/cpm.py

```diff
@@ -112,4 +115,19 @@ def process_questionnaire_response(item, data_dict):
             elif service["text"] == "Owner":
                 data_dict["nhsAsClient"] = [service["answer"][0].get("valueString", "")]
                 data_dict["nhsIdCode"] = service["answer"][0].get("valueString", "")
-    return data_dict
\ No newline at end of file
+    return data_dict
+
+def _raise_invalid_query_params_error():
+        org_code = f'{ORG_CODE_QUERY_PARAMETER_NAME}={ORG_CODE_FHIR_IDENTIFIER}|value'
+        party_key = f'{IDENTIFIER_QUERY_PARAMETER_NAME}={PARTY_KEY_FHIR_IDENTIFIER}|value'
+        service_id = f'{IDENTIFIER_QUERY_PARAMETER_NAME}={SERVICE_ID_FHIR_IDENTIFIER}|value'
+
+        raise tornado.web.HTTPError(
+            status_code=400,
+            log_message=f"Missing or invalid query parameters. "
+                        f"Should one of following combinations: ["
```

Comment:
Just a tiny observation, error message missing 'be' and 'the' - 'Should be one of the following combinations:'

Reply:
Yes, but it matches the current functionality and I don't want to change it just in case an app is expecting it.